dali/test/python/test_operator_decoders_image.py

```diff
@@ -195,28 +159,36 @@ def run_decode_fused(test_fun, path, img_type, batch, device, threads, validatio
 def test_image_decoder_fused():
     threads = 4
     batch_size = 10
-    for test_fun in [create_decoder_slice_pipeline, create_decoder_crop_pipeline, create_decoder_random_crop_pipeline]:
-        # before CUDA 11.4 HW decoder API doesn't support ROI so we get slightly different results HW decoder + slice vs
-        # fused which in this case is executed by the hybrid backend
-        if test_fun == create_decoder_random_crop_pipeline or nvidia.dali.backend.GetNvjpegVersion() < 11040:
-            # random_resized_crop can properly handle border as it has pixels that are cropped out, while
-            # plain resize following image_decoder_random_crop cannot do that and must duplicate the border pixels
-            validation_fun = lambda x, y: np.mean(np.abs(x - y) < 0.5)
+    for test_fun in [
+        create_decoder_slice_pipeline,
+        create_decoder_crop_pipeline,
+        create_decoder_random_crop_pipeline
+    ]:
+        # before CUDA 11.4 HW decoder API doesn't support ROI so we get slightly different results
+        # HW decoder + slice vs fused which in this case is executed by the hybrid backend
+        if test_fun == create_decoder_random_crop_pipeline or \
+                nvidia.dali.backend.GetNvjpegVersion() < 11040:
+            # random_resized_crop can properly handle border as it has pixels that are cropped out,
+            # while plain resize following image_decoder_random_crop cannot do that
+            # and must duplicate the border pixels
+            validation_fun = lambda x, y: np.mean(np.abs(x - y) < 0.5)  # noqa: E731
```

Comment:
```suggestion
            def mean_close(x, y):
                return np.mean(np.abs(x - y) < 0.5)
            validation_fun = mean_close
```

Reply:
(it's merely a suggestion though), same for the else branch.

Reply:
Done